hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -747,4 +776,22 @@ private HapiTokenAirdrop airdropFT(String tokenName, String sender, String recei
         return tokenAirdrop(moving(amountToMove, tokenName).between(sender, receiver))
                 .payingWith(sender);
     }
+
+    private SpecOperation createHollowAccountByFunToken(String hollowAcnt, String token, String owner) {
```

Comment:
There is a createHollow() in UtilVerbs.java

Reply:
You can use the 
```
public static SpecOperation createHollow(
            final int n,
            @NonNull final IntFunction<String> nameFn,
            @NonNull final Function<Address, HapiCryptoTransfer> creationFn)
```
method and pass tokenTransfer for creationFn

Reply:
That method creates more overhead than needed in this situation.